src/generic_invert.jl

```diff
@@ -1,4 +1,9 @@
-export invert 
+export invert, OptimInterface, OptimOptim, OptimNextGen
+using Infiltrator
```

Comment:
Remove using Infiltrator and all @infilrate macros

Reply:
sure